packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1319,4 +1319,30 @@ describe('build', () => {
       'edge.func'
     );
   });
+
+  describe('with Vercel Speed Insights', () => {
```

Comment:
**question:** is this describe useful? there is no `before` nor `beforeAll`, and I don't understand how we're enabling anything.